src/main/java/ro/andreiciortea/yggdrasil/template/acta4h/Curtains.java

```diff
@@ -0,0 +1,60 @@
+package ro.andreiciortea.yggdrasil.template.acta4h;
+
+import io.vertx.core.http.HttpMethod;
+
+import ro.andreiciortea.yggdrasil.template.annotation.RequestMapping;
+import ro.andreiciortea.yggdrasil.template.annotation.Artifact;
+import ro.andreiciortea.yggdrasil.template.annotation.ObservableProperty;
+import ro.andreiciortea.yggdrasil.template.annotation.RdfAddition;
+
+
+@Artifact(type = "Curtains", additions =
+  @RdfAddition(predicates ={"http://www.w3.org/1999/02/22-rdf-syntax-ns#type"}, objects = {"td:Thing"})
+)
+public class Curtains {
+
+  @ObservableProperty
+  public double Energie_Partielle_Volets_Roulants = 0;
+
+  @ObservableProperty
+  public double Energie_Totale_Volets_Roulants = 0;
+
+  @ObservableProperty
+  public double Intensite_Volets_Roulants = 0;
+
+  @ObservableProperty
+  public double Puissance_Volets_Roulants = 0;
+
+  @ObservableProperty
+  public double Tension_Volets_Roulants = 0;
+
+  @RequestMapping(httpMethod = HttpMethod.PUT, path = "/Energie_Partielle_Volets_Roulants")
+  public double setEnergie_Partielle_Volets_Roulants(double energie_Partielle_Volets_Roulants) {
```

Comment:
Is this method naming style required for any reason? If not, let's stick to the standard Java code style. The comment is applicable to all artifact templates.

Reply:
No, this has been introduced by Simon Bienz. I assume he used the same designators that were used in the real-world-data. Will change that.